modules/cli/src/main/scala/scala/cli/commands/export0/Export.scala

```diff
@@ -148,7 +148,19 @@ object Export extends ScalaCommand[ExportOptions] {
     val inputs = options.shared.inputs(args.all).orExit(logger)
     CurrentParams.workspaceOpt = Some(inputs.workspace)
     val baseOptions =
-      initialBuildOptions.copy(mainClass = options.mainClass.mainClass.filter(_.nonEmpty))
+      initialBuildOptions
+        .copy(
+          scalaNativeOptions = initialBuildOptions.scalaNativeOptions.copy(
+            maxDefaultNativeVersions =
```

Comment:
Don't you need in such places to also include the previous `maxDefaultNativeVersions`? Or is it one of those places, where it was empty before anyway?

Reply:
good catch!
it'd be an edge case for it to be non empty, but it's worth including it, lemme fix it

Reply:
fixed